### ISSUE TITLE
TooltipIcon: High Contrast

### ISSUE THREAD

**Issue by ZrianinaMariia** [carbon-design-system/carbon#4783]
## Environment

> Operating system

MacOS Catalina 10.15.1 

> Browser

Firefox 70.0.1 

> Automated testing tool and ruleset

> Assistive technology used to verify

High Contrast mode in Firefox (required test by HP team)

## Detailed description

> What version of the Carbon Design System are you using?

Carbon 10

carbon-components 10.7.3
carbon-components-react 7.7.3

> What did you expect to happen?

I expect to be able to read the tooltip text and to see the background (or borders) when High Contrast mode in Firefox is on (white on black). I expect TooltipIcon to have proper caret and to be able to see the background (or borders) when High Contrast mode in Firefox is on (black on white).

> What happened instead?

High Contrast mode in Firefox is on (white on black) - can not read the text, can not see borders:

![image](https://user-images.githubusercontent.com/5481483/69707660-76d8a780-10fa-11ea-8e4a-864d4e5e295d.png)

High Contrast mode in Firefox is on (black on white) - caret does not looks the same as in normal mode and can not see borders:

![image](https://user-images.githubusercontent.com/5481483/69707628-688a8b80-10fa-11ea-89a9-006b85e58162.png)

> What WCAG 2.1 checkpoint does the issue violate?

e.g. 1.4.3 
https://www.ibm.com/able/guidelines/ci162/contrast.html

## Steps to reproduce the issue

1. Open Firefox preferences 
2. Search for 'advanced'
3. Click on 'Colors'
4. 
    - Set `text` to white and `background` to black to test white on black high contrast mode.
    - Set `text` to black and `background` to white to test black on white high contrast mode.

> Please create a reduced test case in CodeSandbox

https://codesandbox.io/s/codesandbox-4mt04

**Comment by joshblack** [comment]
cc @dakahn if you have a chance to confirm!

**Comment by dakahn** [comment]
@ZrianinaMariia Thanks for reporting. We don't support this method of high contrast color inversion on Firefox in macOS.

If a user wants high contrast support for Firefox in macOS they should navigate to the Accessibility options screen and check one or both of options **Invert Colors** and **Increase Contrast**.
